ml-agents/mlagents/trainers/learn.py

```diff
@@ -73,6 +78,12 @@ def run_training(sub_id: int, run_seed: int, run_options, process_queue):
             docker_target_name=docker_target_name
         )
 
+    sampler = None
```

Comment:
nit: rename to `sampler_config`. `sampler` implies it's a `Sampler` (which it's not)

Reply:
Can this and lesson_config default to empty dictionaries instead? That might simplify null handling later.

Reply:
Sorry, didn't realize that `lesson_config` was actually a LessonController (which means it has a bad name!).